src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -380,4 +380,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Warning,
         summary: "Removed `src/tools/rls` tool as it was deprecated long time ago.",
     },
+    ChangeInfo {
+        change_id: 137081,
+        severity: ChangeSeverity::Info,
```

Comment:
```suggestion
        severity: ChangeSeverity::Warning,
```
To make this a bit more visible, this is a big change.